pkgs/os-specific/linux/systemd/0001-Start-device-units-for-uninitialised-encrypted-devic.patch

```diff
@@ -1,19 +1,24 @@
-From dd2ec741aaa7c587eb7719bbf4b305fe28168b77 Mon Sep 17 00:00:00 2001
+From 3add74855015da7070784e448748efbe6afcb252 Mon Sep 17 00:00:00 2001
 From: Eelco Dolstra <eelco.dolstra@logicblox.com>
 Date: Tue, 8 Jan 2013 15:46:30 +0100
-Subject: [PATCH 01/18] Start device units for uninitialised encrypted devices
+Subject: [PATCH 01/19] Start device units for uninitialised encrypted devices
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
 
 This is necessary because the NixOS service that initialises the
 filesystem depends on the appearance of the device unit.  Also, this
 makes more sense to me: the device is ready; it's the filesystem
 that's not, but taking care of that is the responsibility of the mount
 unit.  (However, this ignores the fsck unit, so it's not perfect...)
+
+Signed-off-by: Jörg Thalheim <joerg@thalheim.io>
```

Comment:
These should probably not be here.

Reply:
Fixed for the future by using: https://github.com/NixOS/nixpkgs/pull/118202/files#diff-a284772b505bd573f834b58591d380128545257d9e02601dbe112548526c8710R132